### PR TITLE
Build Haskell specs in Hydra

### PR DESCRIPTION
**Pull request by nc6**
Testing this for bors...

### CODE REVIEW

**Comment by nc6** [comment]
bors try

**Comment by nc6** [comment]
bors try

**Comment by nc6** [comment]
bors try

File: release.nix [hunk 1]
```diff
@@ -4,7 +4,12 @@ in pkgs.lib.fix (self: {
 
   required = pkgs.lib.hydraJob (pkgs.releaseTools.aggregate {
     name = "cardano-chain-required-checks";
-    constituents = with self; [ byronLedgerSpec byronChainSpec semanticsSpec ];
+    constituents = with self;
+      [ byronLedgerSpec byronChainSpec semanticsSpec
+        pkgs.haskell.packages.ghc861.small-steps
```
Comment (cleverca22) [inline release.nix]: this must name other things within the jobset
so you need to add those to the top-level of release.nix or it wont build them

**Review by cleverca22 — DISMISSED** [review]

File: release.nix (hunk 1 above)
Comment (nc6) [reply release.nix]: Thanks, trying this now!

**Comment by nc6** [comment]
@cleverca22 Does this look ok now?

**Review by ruhatch — APPROVED** [review]
Looks good :)

**Pull request merged**